lib/fuel-auth.js

```diff
@@ -31,9 +35,15 @@ module.exports = class FuelAuth {
 		this.clientSecret = options.clientSecret;
 		this.expiration = null;
 		this.refreshToken = options.refreshToken;
-		this.scope = options.scope;
 		this.version = version;
 		this.globalReqOptions = options.globalReqOptions || {};
+		if(options.authOptions){
+			this.accountId = options.authOptions.accountId;
+			this.authVersion = options.authOptions.authVersion;
+			this.scope = options.authOptions.scope;
+			this.soapUrl = null;
```

Comment:
Are this.soapUrl and this.restUrl read from other modules?

Reply:
Just added for future purpose.